templates/pages/index.html.njk

```diff
@@ -34,14 +34,13 @@
   <section class="c-content">
     <div class="l-container">
       <h2 class="fz--beta">
-        Call for Artists is open!
+        Make a scholarship donation!
```

Comment:
Don’t we want to apply the same titlecase rule as for other headlines here?

Reply:
Yup I'd use titlecase too
`Make a Scholarship Donation`